src/main/java/org/openrewrite/java/testing/junit5/RemoveTryCatchBlocksFromUnitTests.java

```diff
@@ -0,0 +1,120 @@
+
+    @Override
+    public String getDisplayName() {
+        return "Unit test should throw exceptions instead of using `try-catch` blocks";
+    }
+
+    @Override
+    public String getDescription() {
+        return "When the code under test in a unit test throws an exception, the test itself fails. " +
+                "Therefore, there is no need to surround the tested code with a try-catch.";
+    }
+
+    @Override
+    public Set<String> getTags() {
+        return Collections.singleton("RSPEC-3658");
+    }
+
+    @Override
+    public TreeVisitor<?, ExecutionContext> getVisitor() {
+        return new RemoveTryCatchBlocksFromUnitsTestsVisitor();
+    }
+
+    private static class RemoveTryCatchBlocksFromUnitsTestsVisitor extends JavaIsoVisitor<ExecutionContext> {
+        @Override
+        public J.MethodDeclaration visitMethodDeclaration(J.MethodDeclaration md, ExecutionContext ctx) {
+            // check if method is Unit test
+            List<J.Annotation> annotations = md.getAllAnnotations();
+            if (annotations.stream().noneMatch(ann -> MATCHERS.stream().anyMatch(matcher -> matcher.matches(ann)))) {
+                return md;
+            }
+
+            if (md.getBody() == null) {
+                return md;
+            }
+            // find try catch block
+            List<Statement> statements = md.getBody().getStatements();
+            for (Statement s : statements) {
+                if (s instanceof J.Try) {
+                    List<J.Try.Catch> catch_ = ((J.Try) s).getCatches();
+                    for (J.Try.Catch c : catch_) {
+                        J.Block block = c.getBody();
+                        List<Statement> catchStatements = block.getStatements();
+                        for (Statement st : catchStatements) {
+                            if (st instanceof J.MethodInvocation) {
+                                if (ASSERT_FAIL_MATCHER.matches((J.MethodInvocation) st)) {
+                                    getCursor().putMessage("KEY", md);
+                                }
+                            }
+                        }
+                    }
+                    break;
+                }
+            }
+
+            return super.visitMethodDeclaration(md, ctx);
+        }
+
+        public J.MethodInvocation build(J.MethodInvocation mi) {
+            Expression argument = mi.getArguments().get(0);
+            if (mi.getArguments().get(0) instanceof J.MethodInvocation){
+                argument = ((J.MethodInvocation) mi.getArguments().get(0)).getSelect();
+            }
+
+            maybeRemoveImport("org.junit.Assert");
+            maybeAddImport("org.junit.jupiter.api.Assertions");
+            return JavaTemplate.builder("Assertions.assertDoesNotThrow(#{any()})")
+                    .staticImports("org.junit.jupiter.api.Assertions")
```

Comment:
```suggestion
                    .imports("org.junit.jupiter.api.Assertions")
```

Reply:
What you are also missing here is a call to `javaParser()` to make it aware of the JUnit Jupiter types:

```java
javaParser(JavaParser.fromJavaVersion().classpathFromResources(ctx, "junit-jupiter-api-5.9"))
```

Please also be aware that `assertDoesNotThrow()` works differently: It is used to wrap a code block. Typically you would let it wrap a lambda containing the statements which you here have in the try-block. See https://junit.org/junit5/docs/5.8.2/api/org.junit.jupiter.api/org/junit/jupiter/api/Assertions.html#assertDoesNotThrow(org.junit.jupiter.api.function.Executable) for details.